spec/results/proto-aws-s3-blobstore-iam-profile.yml

```diff
@@ -8,17 +8,17 @@ cloud_provider:
     agent:
       mbus: https://mbus:<!{meta.vault}/mbus_bootstrap:password!>@0.0.0.0:6868
     aws:
-      access_key_id: <!{meta.vault}/aws:access_key!>
```

Comment:
this seems like an a fix of an error that was there before right?

Reply:
access keys should not be used when using iam profile. so yes this is a fix